src/main/java/seedu/address/logic/commands/ListPeopleCommand.java

```diff
@@ -8,9 +8,9 @@
 /**
  * Lists all persons in the address book to the user.
  */
-public class ListCommand extends Command {
+public class ListPeopleCommand extends Command {
```

Comment:
Note:
- What purpose does this command actually serve to the user, since the main page already lists all the contacts in the addressbook?